lib/Mojolicious/Command/Author/generate/app.pm

```diff
@@ -196,7 +196,7 @@ done_testing();
 </p>
 
 @@ config
-% use Mojo::Util qw(sha1_sum steady_time);
+% use Mojo::Util qw(generate_secret);
 ---
 secrets:
-  - <%= sha1_sum $$ . steady_time . rand  %>
+  - <%= generate_secret() %>
```

Comment:
The generated secret remains insecure unless CryptX is installed.

This would be fixed by #2200.

Reply:
And we are ok with that.

Reply:
That invalidates Mojolicious as a valid framework to use in the EU as NIS2 requires secure-by-default and no predefines secrets or credentials allowed.

Reply:
@abraxxa No.

Reply:
@abraxxa The burden of proof is on you. If you make a handwavy claim you get a handwavy rebuttal from me. :)

Reply:
If you search https://eur-lex.europa.eu/legal-content/EN/TXT/HTML/?uri=CELEX:32022L2555 for 'by default' you see the points I'm referring to, (98) and (104).

Reply:
@abraxxa I don't see how any of that applies to the situation here. So I'm just going to assume you've misunderstood the feature.

Reply:
And on that note we should end the topic before things get out of hand.